modules/ducktests/tests/ignitetest/services/utils/ignite_aware.py

```diff
@@ -172,6 +172,7 @@ def init_persistent(self, node):
 
     def _prepare_configs(self, node):
         config = self.config.prepare_for_env(test_globals=self.globals, node=node, cluster=self)
+        config = self.spec.extend_config(test_globals=self.globals, node=node, cluster=self)
```

Comment:
This should be executed **before** prepare_for_env.
globals should have higher priority than spec.

Reply:
We should have config as parameter of `extend_config` method, also.